src/main/resources/application.properties

```diff
@@ -1,6 +1,7 @@
+server.PORT=7700
 spring.datasource.url=jdbc:mysql://localhost:3306/hr
 spring.datasource.username=root
-spring.datasource.password=password
+spring.datasource.password=system
```

Comment:
Do not change the default one, always use variables - I will demonstrate that in next class